pkg/cover/html.go

```diff
@@ -156,23 +155,19 @@ func (rg *ReportGenerator) DoRawCoverFiles(w http.ResponseWriter, progs []Prog,
 	uniquePCs := make(map[uint64]bool)
 	for _, prog := range progs {
 		for _, pc := range prog.PCs {
-			if uniquePCs[pc] {
-				continue
+			if !uniquePCs[pc] {
+				uniquePCs[pc] = true
+				pcs = append(pcs, pc)
 			}
-			uniquePCs[pc] = true
-			pcs = append(pcs, pc)
 		}
 	}
 	sort.Slice(pcs, func(i, j int) bool {
 		return pcs[i] < pcs[j]
 	})
-	sort.Slice(rg.Frames, func(i, j int) bool {
```

Comment:
Below we calls sort.Search on it, so it must be sorted. Is it sorted before this function is called?

Reply:
move the sort into pkg/cover/report.go so rg.Frames can be used w/ also sorted data.